src/showplanner/ProModeButtons.tsx

```diff
@@ -15,7 +15,10 @@ export default function ProModeButtons({ channel }: { channel: number }) {
 
   return (
     <>
-      <div className="row m-0 p-1 card-header channelButtons proMode">
+      <div className="row m-0 p-1 card-header channelButtons proMode hover-menu">
+        <span className="hover-label">
+          ProMode
```

Comment:
```suggestion
          ProMode™
```

NEVER:clap:NOT:clap:ON:clap:BRAND:clap:!

Reply:
Done